pkg/subctl/cmd/deploybroker.go

```diff
@@ -42,6 +46,13 @@ func init() {
 	deployBroker.PersistentFlags().BoolVar(&disableDataplane, "no-dataplane", true,
 		"Don't install the Submariner dataplane on the broker (default)")
 	lighthouse.AddFlags(deployBroker, "service-discovery")
+
+	deployBroker.PersistentFlags().BoolVar(&globalnetEnable, "globalnet-enable", false,
```

Comment:
One option is to get rid of this flag and depend on globalnet-cidr-range alone (i.e., if its present, it means GN is enabled otherwise not). What's your thought on this?
CC: @tpantelis @mangelajo @skitt @mkolesnik

Reply:
Discussed with @vthapar in today's sync call and he suggested that by having globalnet-enable, we can avoid asking the user for globalnet-cidr-range/globalnet-cluster-size as inputs if the user is ok with the default values, which is a good point.

Reply:
This is for cases when user just wants to enable globalnet without having to manually specify any CIDR.

Reply:
Similar to displaying default globalnet-cidr-range and globalnet-cluster-size can you also display that by default globalnet-enable is false.

Current output:
Usage:
  subctl deploy-broker [flags]

Flags:
      --broker-cluster-context string      Broker cluster context
      --clustercidr string                 cluster CIDR
      --clusterid string                   cluster ID used to identify the tunnels
      --colorcodes string                  color codes (default "blue")
      --dataplane                          Install the Submariner dataplane on the broker
      --disable-nat                        Disable NAT for IPsec
      --globalnet-cidr-range string        Global CIDR supernet range for allocating GlobalCIDRs to each cluster (default "169.254.0.0/16")
      --globalnet-cluster-size uint        Default cluster size for GlobalCIDR allocated to each cluster (default 8192)
      --globalnet-enable                   Enable support for Overlapping CIDRs in connecting clusters
  -h, --help                               help for deploy-broker
      --ikeport int                        IPsec IKE port (default 500)
      --ipsec-debug                        Enable IPsec debugging (verbose logging)

Reply:
It is picks default from parameters passed and looks like it doesn't show boolean defaults for anything.

Reply:
You can add it as part of the description. i.e., "Enable support for Overlapping CIDRs in connecting clusters (default disabled)"

Reply:
Done